src/main/java/org/folio/circulation/services/LostItemFeeChargingService.java

```diff
@@ -174,13 +174,12 @@ private String getFeeFineActionCreatedAt(ReferenceDataContext context) {
 
   private boolean shouldChargeItemFee(LostItemPolicy policy) {
     // Set cost fee is only supported now
-    return policy.getChargeAmountItem().getChargeType() == ChargeAmountType.SET_COST
-      && isGreaterThanZero(policy.getChargeAmountItem().getAmount());
+    return isGreaterThanZero(policy.getSetCostChargeFee().getAmount());
   }
 
   private boolean shouldChargeProcessingFee(LostItemPolicy policy) {
     return policy.shouldChargeProcessingFee()
-      && isGreaterThanZero(policy.getLostItemProcessingFee());
+      && isGreaterThanZero(policy.getLostItemProcessingFee().getAmount());
```

Comment:
Could this be moved to the fee? Maybe the fee could have a `isChargeable` method?

Reply:
Good point, moved.